src/Sulu/Bundle/ContentBundle/Resources/public/js/components/content/form/main.js

```diff
@@ -179,6 +179,10 @@ define(['sulucontent/components/content/preview/main'], function(Preview) {
             this.createForm(this.data).then(function() {
                 this.initializeResourceLocator();
                 this.changeTemplateDropdownHandler();
+
+                if (!!Config.get('sulu-content').collaboration.enabled) {
```

Comment:
also here we have a feature in the ResourceBundle called feature-flags see docs for filters http://docs.sulu.io/en/latest/cookbook/integrating-custom-filters.html